### PR TITLE
obisObserver: add commands

### PR DESCRIPTION
**Pull request by aleks-and**

### CODE REVIEW

File: src/analog/utils/time.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {BrandType} from '../../types.js';
  * Time `2023-04-03T14:01:17.000Z` is `1680530477` in Unix time representation.
  * Unix time for year `2000` is `946684800` so seconds since year `2000` is `1680530477` - `946684800` = `733845677`.
  */
-export type TTime2000 = BrandType<number, 'seconds'>;
+export type TTime2000 = TUint32;
```
Comment (DarkPark) [inline src/analog/utils/time.ts:11]: nope

this will ruin the docs

File: src/obis-observer/commands/downlink/getArchiveState.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
+/**
+ * Downlink command.
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/getArchiveState.ts:2]: would be nice to copy-paste commands description from jooby-docs

here and in other places

File: src/obis-observer/commands/downlink/getMeterDate.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+ *     requestId: 4,
+ *     meterId: 3
+ * };
+ *
+ * const bytes = getMeterDate.toBytes(parameters);
+ *
+ * // command binary representation
+ * console.log(bytes);
+ * // output:
+ * // [122, 5, 4, 0, 0, 0, 3]
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/getMeterDate.ts:20]:
```suggestion
 * [122, 5, 4, 0, 0, 0, 3]
```

File: src/obis-observer/commands/downlink/getMeterId.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+ * @example
+ * ```js
+ * import * as getMeterId from 'jooby-codec/obis-observer/commands/downlink/getMeterId.js';
+ *
+ * const parameters = {
+ *     requestId: 0,
+ *     address: 'ma2375'
+ * };
+ *
+ *  * const bytes = getMeterId.toBytes(parameters);
```
Comment (DeadByElpy) [inline src/obis-observer/commands/downlink/getMeterId.ts:15]:
```suggestion
 *  const bytes = getMeterId.toBytes(parameters);
```

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: src/analog/utils/time.ts (hunk 1 above)
Comment (aleks-and) [reply src/analog/utils/time.ts]: fixed

File: src/obis-observer/commands/downlink/getArchiveState.ts (hunk 2 above)
Comment (aleks-and) [reply src/obis-observer/commands/downlink/getArchiveState.ts]: done

File: src/obis-observer/commands/downlink/getMeterDate.ts (hunk 3 above)
Comment (aleks-and) [reply src/obis-observer/commands/downlink/getMeterDate.ts]: fixed

File: src/obis-observer/commands/downlink/getMeterId.ts (hunk 4 above)
Comment (aleks-and) [reply src/obis-observer/commands/downlink/getMeterId.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**